packages/react-scripts/scripts/init.js

```diff
@@ -213,10 +231,23 @@ module.exports = function(
     }
   }
 
-  if (useTypeScript) {
+  if (args.find(arg => arg.includes('typescript'))) {
+    console.log();
```

Comment:
Intentional log?

Reply:
Yes, for spacing. I don't like it necessarily, but it's the convention in that file. We can change in v4 methinks.